SwiftReflector/Demangling/TLDefinition.cs

```diff
@@ -232,5 +232,15 @@ public TLBaseConformanceDescriptor (string mangledName, SwiftName module, SwiftC
 		}
 		public SwiftClassType ProtocolRequirement { get; private set; }
 	}
+
+	public class TLAssociatedTypeDescriptor : TLClassElem {
+		public TLAssociatedTypeDescriptor (string mangledName, SwiftName module, SwiftClassType protocol, SwiftName associatedTypeName, ulong offset)
+			: base (CoreCompoundType.AssociatedTypeDescriptor, mangledName, module, protocol, offset)
+		{
+			AssociatedTypeName = associatedTypeName;
+		}
+
+		public SwiftName AssociatedTypeName { get; private set; }
+	}
```

Comment:
Little trick to type less:

```csharp
public class TLAssociatedTypeDescriptor : TLClassElem {
		public TLAssociatedTypeDescriptor (string mangledName, SwiftName module, SwiftClassType protocol, SwiftName associatedTypeName, ulong offset)
			: base (CoreCompoundType.AssociatedTypeDescriptor, mangledName, module, protocol, offset)
				=> AssociatedTypeName = associatedTypeName;

		public SwiftName AssociatedTypeName { get; }
	}
```
Don't ask me when it was added in the language, but is nice for getter only properties.

Reply:
👍